firefox-ios/Client/Frontend/Library/HistoryPanel/HistoryPanel.swift

```diff
@@ -368,56 +368,39 @@ class HistoryPanel: UIViewController,
 
```

Comment:
small improvement to line 367 if you don't mind changing it, it can be shortened to `guard let self else { return nil }`